tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -1998,7 +1998,7 @@ public function dataInstanceOf(): array
 	{
 		return [
 			[
-				'PhpParser\Node\Expr\ArrayDimFetch',
+				'*NEVER*',
```

Comment:
This was a test for instanceof inside `while`, so should be replaced with something that actually tests this condition.